website/docs/r/external_network_v2.html.markdown

```diff
@@ -70,6 +70,60 @@ resource "vcd_external_network_v2" "ext-net-nsxt" {
 }
 ```
 
+## Example Usage (NSX-T Segment backed External Network with a Direct Org VDC network [only VCD 10.3+])
+
+```hcl
+data "vcd_nsxt_manager" "main" {
+  name = "nsxManager"
+}
+
+resource "vcd_external_network_v2" "ext-net-nsxt-segment" {
+  name        = "nsxt-external-network"
+  description = "First NSX-T segment backed network"
+
+  nsxt_network {
+    nsxt_manager_id   = data.vcd_nsxt_manager.main.id
+    nsxt_segment_name = "existing-nsxt-segment"
```

Comment:
Maybe let's add a comment above this line explaining more about this field. I think it would be easier for the user  to notice the difference from the example above when just scanning the examples, as they are very similar.

Reply:
Agreed. I have put a note right in this section. Here is how it looks in docs:

![image](https://user-images.githubusercontent.com/15804230/133990857-d963d339-29d4-465e-a685-4df5c4c74cd1.png)